k8s/persistent_volumes.go

```diff
@@ -0,0 +1,42 @@
+package k8s
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func GetPersistentVolumeClaim(ctx context.Context, name, namespaceOverride string) (*corev1.PersistentVolumeClaim, error) {
```

Comment:
Is this function used anywhere?

Reply:
At the moment it is not, but most likely will be used in follow up code. I can remove it, if you prefer so.

Reply:
It's OK if it will be used soon.